src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -100,16 +101,21 @@ public static ControlFlowGraph GetControlFlowGraph(Operations.IBlockOperation bo
 
             if (body.Parent != null)
             {
-                // PROTOTYPE(dataflow): Special message?
-                throw new ArgumentException();
+                throw new ArgumentException(CodeAnalysisResources.NotARootOperation, nameof(body));
```

Comment:
>ArgumentException [](start = 26, length = 17)

Makes me wonder whether the checking for non-null input and non-null parent and non-null model could be pulled into a common helper.

Reply:
Sure, I refactor by passing in the argument name for the exceptions into the common helper.